applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -620,28 +622,34 @@ gateway_to_endpoint(DestinationNumber
              ,{<<"Matched-Number">>, DestinationNumber}
               | gateway_from_uri_settings(Gateway)
              ]),
-    kz_json:from_list(
-      props:filter_empty(
-        [{<<"Route">>, gateway_dialstring(Gateway, DestinationNumber)}
-        ,{<<"Callee-ID-Name">>, kz_term:to_binary(DestinationNumber)}
-        ,{<<"Callee-ID-Number">>, kz_term:to_binary(DestinationNumber)}
-        ,{<<"To-DID">>, kz_term:to_binary(DestinationNumber)}
-        ,{<<"Invite-Format">>, InviteFormat}
-        ,{<<"Caller-ID-Type">>, CallerIdType}
-        ,{<<"Bypass-Media">>, BypassMedia}
-        ,{<<"Codecs">>, Codecs}
-        ,{<<"Auth-User">>, Username}
-        ,{<<"Auth-Password">>, Password}
-        ,{<<"Custom-SIP-Headers">>, SipHeaders}
-        ,{<<"SIP-Interface">>, SipInterface}
-        ,{<<"Endpoint-Type">>, EndpointType}
-        ,{<<"Endpoint-Options">>, EndpointOptions}
-        ,{<<"Endpoint-Progress-Timeout">>, kz_term:to_binary(ProgressTimeout)}
-        ,{<<"Custom-Channel-Vars">>, kz_json:from_list(CCVs)}
-        ,{<<"Outbound-Caller-ID-Number">>, CIDNumber}
-        ,{<<"Outbound-Caller-ID-Name">>, CIDName}
-         | maybe_get_t38(Gateway, OffnetJObj)
-        ])).
+    PropList = props:filter_empty(
+                 [{<<"Route">>, gateway_dialstring(Gateway, DestinationNumber)}
+                 ,{<<"Callee-ID-Name">>, kz_term:to_binary(DestinationNumber)}
+                 ,{<<"Callee-ID-Number">>, kz_term:to_binary(DestinationNumber)}
+                 ,{<<"To-DID">>, kz_term:to_binary(DestinationNumber)}
+                 ,{<<"Invite-Format">>, InviteFormat}
+                 ,{<<"Caller-ID-Type">>, CallerIdType}
+                 ,{<<"Bypass-Media">>, BypassMedia}
+                 ,{<<"Codecs">>, Codecs}
+                 ,{<<"Auth-User">>, Username}
+                 ,{<<"Auth-Password">>, Password}
+                 ,{<<"Custom-SIP-Headers">>, SipHeaders}
+                 ,{<<"SIP-Interface">>, SipInterface}
+                 ,{<<"Endpoint-Type">>, EndpointType}
+                 ,{<<"Endpoint-Options">>, EndpointOptions}
+                 ,{<<"Endpoint-Progress-Timeout">>, kz_term:to_binary(ProgressTimeout)}
+                 ,{<<"Custom-Channel-Vars">>, kz_json:from_list(CCVs)}
+                 ,{<<"Outbound-Caller-ID-Number">>, CIDNumber}
+                 ,{<<"Outbound-Caller-ID-Name">>, CIDName}
+                  | maybe_get_t38(Gateway, OffnetJObj)
+                 ]) ++ maybe_rtcp_mux(RTCP_MUX),
```

Comment:
@lazedo i cannot add as regular property to `Custom-Channel-Var` [here](https://github.com/sergey-safarov/kazoo/blob/ea272da70b547ed70f08e5abe23539aff4e5d97e/applications/stepswitch/src/stepswitch_resources.erl#L620-L624). Because this feature is enabled in FreeSwitch by default and `<<"RTCP-MUX">>` property with `true` value is useless and with `false` value is been removed by `props:filter_empty`.
I can simple move `++ maybe_rtcp_mux(RTCP_MUX)` to string [624](https://github.com/sergey-safarov/kazoo/blob/ea272da70b547ed70f08e5abe23539aff4e5d97e/applications/stepswitch/src/stepswitch_resources.erl#L624). Is this will be correct?